src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,46 @@
-export const Tabs = () => {};
+import { FC } from 'react';
+import { Tab } from '../../types/tab';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void;
+};
+
+export const Tabs: FC<Props> = ({ tabs, selectedTabId, onTabSelected }) => {
+  const selected = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  const handleClick = (tab: Tab) => {
+    if (selected.id !== tab.id) {
+      onTabSelected(tab);
+    }
+  };
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              className={selected.id === tab.id ? 'is-active' : ''}
```

Comment:
it's better to use classnames especially when one of your option in ternary operator is empty